src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectMessage.cs

```diff
@@ -128,6 +128,20 @@ public OpenIdConnectMessage(IEnumerable<KeyValuePair<string, string[]>> paramete
             }
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OpenIdConnectMessage"/> class.
+        /// </summary>
+        /// <param name="json">the json object from which the instance is created.</param>
```

Comment:
Nit: the -> The; json -> JSON.

Reply:
Thanks! Grammar fixes are appreciated.

[Fixed here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1331/commits/1db9d409a7866b1785344ec54802cf5c1e0ccef7#diff-9018aeed52729e8285f021fcba37a1f0).